contracts/TestUtils.sol

```diff
@@ -0,0 +1,52 @@
+pragma solidity ^0.4.23;
+
+// Copyright 2017 OpenST Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// ----------------------------------------------------------------------------
+// Common: TestingUtils
+//
+// http://www.simpletoken.org/
+//
+// ----------------------------------------------------------------------------
+
+/**
+ *	@title TestingUtils 
+ *
+ *	@notice contains functions used for testing on Mock contracts. 
+ */
+library TestUtils {
+
+	/**
```

Comment:
I think the names/descriptions require revision.

The function is called `mapStorageKey` but it knows nothing about a mapping and could be used for non-mapping purposes.

The `_key` and return values both are described as the "key to the mapping value" even though (a) neither the library nor the function knows anything about a mapping and (b) the return value is a hash of the ABI encoded values of more inputs than the `_key` alone.